test/OpenTelemetry.Instrumentation.Process.Tests/ProcessMetricsTests.cs

```diff
@@ -173,8 +212,6 @@ public void CheckValidGaugeValueWhen2MeterProviderInstancesHaveTheSameMeterName(
 
         meterProviderA.ForceFlush(MaxTimeToAllowForFlush);
 
-        // Note: This fails due to:
```

Comment:
this note is removed intentionally?

Reply:
yes, the test will not fail with the current design